app/src/main/assets/templates/send.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>{% if title %}{{ title }}{% else %}OnionShare{% endif %}</title>
```

Comment:
These `assets` are 1:1 what is in the onionshare repo. I managed to find a way to use them without needing to make a single modification.

Reply:
Nice :-)